timescale-cloud/maintenance.md

```diff
@@ -0,0 +1,51 @@
+# Maintenance
+On Timescale Cloud, software updates are handled automatically by us, and you do not need to perform any actions.
+
+Non-critical software updates are applied during a maintenance window that you can define to suit your workload. If we detect a security vulnerability that affects you, we might need to perform maintenance outside of the scheduled maintenance window.
+
+<highlight type="important">
+During a maintenance update, the DNS name remains the same, but the IP address
+it points to changes.
+</highlight>
+
+## Non-critical maintenance updates
+Non-critical upgrades are made available before the upgrade is performed
+automatically. During this time you can click `Apply upgrades` to start the
+upgrade at any time. However, after the time expires, usually around a week,
+the upgrade is triggered automatically in the next available maintenance window
+for your service. You can configure the maintenance window so that these
+upgrades are started only at a particular time, on a set day of the week. If
+there are no pending upgrades available during a regular maintenance window, no
+changes are performed.
+
+When you are considering your maintenance window schedule, you might prefer to
+choose a day and time that usually has very low activity, such as during the
+early hours of the morning, or over the weekend. This can help minimize the
+impact of a short service interruption. Alternatively, you might prefer to have
+your maintenance window occur during office hours, so that you can monitor your
+system during the upgrade.
+
+### Procedure: Adjusting your maintenance window
+1.  [Log in to your Timescale Cloud account][mst-login]. Click the name of the service that
+    you want to manage the maintenance window for.
+1.  In the `Maintenance window` section, click `Change`.
+1.  In the `Service Maintenance Window` dialog, select the day of the week and
+    the time (in Universal Coordinated Time) you want the maintenance window to
+    start. Maintenance windows can run for up to four hours.
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/mst-maintwindow.png" alt="Adjust maintenance window"/>
+1.  Click `Save Changes`.
+
+## Critical updates
+Critical upgrades and security fixes can be installed outside normal maintenance windows when necessary, and sometimes require have a short outage.
```

Comment:
change "can" to "may": "security fixes can be installed outside normal maintenance windows" -> "security fixes may be installed outside normal maintenance windows". The former implies it is an action that they may take, the latter implies it is something that happens regardless.

Reply:
I'm not sure how you're drawing that difference between "can" and "may" honestly 😂  Generally speaking, they mean exactly the same thing, but "may" is more formal. In my own writing practice, I tend to avoid "may" because it's a bit more ambiguous than some of the alternatives (it can mean that you have permission to do something conferred by someone else, or that you have the ability in yourself to do something), and it just reads a bit too stiff and formal for the writing style I am using here. In this case, I think if you want to make the sentence more assertive, using "are" would work better?

Reply:
Fair enough. I agree what you say, but for some reason when I read "can be installed", I read it as giving the user the option, rather than stating it may be done. With "may be installed" I read it as something that could happen. But I also do get that they are really the same.

:+1: to using "are" instead of "can be"